CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 #### Bug fixes
 
+  + Fix types on named arguments are wrapped incorrectly when preceding comments (#1124) (Guillaume Petiot)
```

Comment:
I'm not sure it is very useful, even the contributor's name is redundant with the PR's number to me, unless you want to grep on someone's name (what for?)

Reply:
I'm not sure this project is about changelogs but I agree, we should follow a standard.

There is https://keepachangelog.com/en/1.0.0/ which only show external contributors and no PR#.
Also, OCaml https://github.com/ocaml/ocaml/blob/trunk/Changes (PR#, contributor, reviewers and sometimes reporter, lots of details and sections)
and Dune https://github.com/ocaml/dune/blob/master/CHANGES.md (PR#, issue#, contributor and some details).

I don't have strong opinions on which format is better. I like OCaml's for the details and the section.

What do you think @emillon and @jberdine ?

Reply:
Having the reviewers makes sense for OCaml given the number of contributors they have, so you know who to blame, it looks like an overkill for ocamlformat for now. But I agree that more details on what a PR does can't hurt.

Reply:
PR number is a must have I'd say because it allows one to dig deeper if they don't understand or want to know all the details about a specifc change.

Iimo, the author name is just a way to credit the contributors but I still like it. I agree it's not as strictly necessary as the PR number(s?).